widgets/MarketoReportingPlugin-batchsource.json

```diff
@@ -0,0 +1,71 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "display-name": "Marketo Reporting",
+  "configuration-groups": [
+    {
+      "label": "General",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Rest API endpoint",
+          "name": "restApiEndpoint"
+        }
+      ]
+    },
+    {
+      "label": "Authentication",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Client ID",
+          "name": "clientId"
+        },
+        {
+          "widget-type": "password",
+          "label": "Client Secret",
+          "name": "clientSecret"
+        }
+      ]
+    },
+    {
+      "label": "Report",
+      "properties": [
+        {
+          "widget-type": "select",
+          "label": "Report Type",
+          "name": "reportType",
+          "widget-attributes": {
+            "default": "leads",
+            "values": [
+              "leads",
+              "activities"
+            ]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Start Date",
+          "name": "startDate"
```

Comment:
please add placeholder with format. Also somebody told me we already have widget to choose dates. Can you check this?

Reply:
Unfortunately date widget still on review. Waiting for it so much... :(